### DISCUSSION TITLE
Newest Skyblock Update

### DISCUSSION THREAD

**Discussion by Myzumi** [HypixelDev/PublicAPI#564] [Ideas]
The new update brought up some new stuff, which isn't in the api yet but would be very useful:

Skyblock Levels
The Perks that are unlocked at x level
Current Level Prefix aswell as unlocked ones
Unlocked backpack slots
Current holding Dungeon runs from the Croesus NPC

This would help developers out more on displaying useful data or giving users benefits on specific stats (Like discord roles or other)

**Comment by MattTheCuber** [comment]
Hello Myzumi,

Don't mean to be critical but a few things in the post could be written better.
First, we only need to a player's Skyblock Experience, nothing more.
We don't need to level prefix because we can calculate their level (unless you meant something else which isn't clear)
Current holding Dungeon runs from the Croesus NPC was already added to the profiles endpoint.

But, yes I would definitely like to see Skyblock Experience in the API as calculating it without museum is impossible lol.

**Reply by Myzumi** [reply]
Hi,

I was talking about the new Skyblock update that came out today, introducing new skyblock levels. As the patchnotes states, there are also "level prefixes" when reaching certain levels.

**Reply by Myzumi** [reply]
Also, this was not my intention to answer this in a thread, this is what github mobile just made me do. I also cannot edit messages here so i cannot merge this one.

**Reply by MattTheCuber** [reply]
No worries. I didn't notice the Skyblock Level prefixes in the patch notes. That's my bad.
